massa-network-exports/src/network_controller.rs

```diff
@@ -450,3 +450,13 @@ impl NetworkManager {
         Ok(())
     }
 }
+
+/// Used by the bootstrap server to run async tasks, allowing the bootstrap module to
+/// remove the tokio dependency.
+pub fn make_runtime() -> tokio::runtime::Runtime {
```

Comment:
** maybe we can remove the tokio rt by using async-std :: block_on

Reply:
that would require adding an async-std dependency.

Once `get_bootstrap_peers` is not async, that would allow this to be removed as well.

Reply:
ok :)